App/Containers/ThreadsList.js

```diff
@@ -86,14 +94,21 @@ class ThreadsList extends React.PureComponent {
       })
     }
     if (
+      this.props.threads &&
```

Comment:
Check that we have an object, a previous object, and that they aren't the same

Reply:
Don't need to check that this exists. See comment below.

Reply:
See my response to that comment. We'll still need to check this, though indirectly now.